Cubical/Foundations/HLevels.agda

```diff
@@ -64,11 +65,26 @@ isContrSigma {A = A} {B = B} (a , p) q =
      , ( λ x i → p (x .fst) i
        , h (p (x .fst) i) (transp (λ j → B (p (x .fst) (i ∨ ~ j))) i (x .snd)) i))
 
+isProp→isPropPathP : (∀ a → isProp (B a))
```

Comment:
Wouldn't it be more natural to prove this using https://github.com/agda/cubical/blob/master/Cubical/Core/Prelude.agda#L172 ?

You get two PathP's, convert them to Path with transp using that lemma and then use isPropB ?

Reply:
I'm not sure what you mean. Wouldn't one need to first show that `toPathP` is an equivalence since applying isPropB to the transported path otherwise can't be pulled back?

Reply:
Ah right, nevermind.